dask/array/tests/test_routines.py

```diff
@@ -104,6 +104,25 @@ def test_dot_method():
     assert_eq(a.dot(b), x.dot(y))
 
 
+@pytest.mark.parametrize('shape, axis', [
+    [(10, 15, 20), 0],
+    [(10, 15, 20), 1],
+    [(10, 15, 20), 2],
+    [(10, 15, 20), -1],
+])
+@pytest.mark.parametrize('n', [
+    0,
+    1,
+    2,
+    3,
+])
```

Comment:
Ok dropped `3`. So it tests the no-op case, the default case, and some non-trivial case (i.e. `2`).